itou/communications/dispatch/email.py

```diff
@@ -30,5 +21,26 @@ def build(self):
         )
 
     def send(self):
+        if (
+            # don't check if the user is still in the organization if it's already a forwarded notification
```

Comment:
```suggestion
            # If it is already a forwarded notification, do not check if the user is still a member of the organization.
```

nit: c'est probablement moi mais j'ai trouvé le premier IF ambigu